s3-source-connector/src/test/java/io/aiven/kafka/connect/s3/source/testutils/ContentUtils.java

```diff
@@ -61,27 +70,30 @@ public static void writeParquetFile(final String tempFilePath, final String name
     private static void writeParquetFile(final String outputPath, final Schema schema, final String name1,
             final int numOfRecords) throws IOException {
 
-        // Create sample records
-        GenericData.Record user;
+        final List<Struct> allParquetRecords = new ArrayList<>();
+        // Write records to the Parquet file
```

Comment:
I don't know if it's worthwhile creating the list versus writing directly to the file!  It doesn't bother me, but this comment should be fixed (we're not writing here).